internal/services/signalr/signalr_service_resource_test.go

```diff
@@ -393,6 +393,21 @@ func TestAccSignalRService_upstreamSetting(t *testing.T) {
 	})
 }
 
+func TestAccSignalRService_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_signalr_service", "test")
+	r := SignalRServiceResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
```

Comment:
This isn't a complete schema, so shouldn't be called `complete` here. Since it's only testing tags, it should be named something like:
```suggestion
func TestAccSignalRService_withTags(t *testing.T) {
	data := acceptance.BuildTestData(t, "azurerm_signalr_service", "test")
	r := SignalRServiceResource{}

	data.ResourceTest(t, r, []acceptance.TestStep{
		{
			Config: r.withTags(data),
```